src/firmware/app/control/robot_controller.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include "firmware/app/control/trajectory_planner.h"
+#include "firmware_new/boards/frankie_v1/io/drivetrain.h"
+#include "firmware_new/boards/frankie_v1/io/drivetrain_unit.h"
+
+typedef struct RobotController RobotController_t;
+
+/**
+ * Initialize the IO layer for the Robot Controller
+ *
+ * NOTE: This does not take ownership of the given drivetrain. The drivetrain must be
+ * destroyed separately.
+ *
+ * @param drivetrain [in] The drivetrain of the robot.
+ */
+RobotController_t* io_robot_controller_create(Drivetrain_t* drivetrain);
```

Comment:
since this is in the `app` folder, all the functions should probably be `app_robot_controller*` ditto rest of the file

Reply:
Whoops, renamed 👍